feat(blog): add blog post about gather & metaverse (#306)
satellytes/satellytes.com: blog-posts/gather-metaverse-bump-each-other-2021.md

---
path: /blog/gather-metaverse-bump-each-other-2021/
date: 2021-12-21
title: How we joined the Metaverse with Gather 
featuredImage: images/gather/teaser.png
seoMetaText: A few weeks ago, we introduced Gather at Satellytes to fight Zoom Fatigue. Soon after, we saw the magic of the Metaverse happening.
teaserText: A few weeks ago, we introduced Gather at Satellytes to make our daily remote life more pleasant and to fight Zoom Fatigue. Soon after, we saw the magic of the Metaverse happening.
attribution:
    creator: Made with Gather Tiles
    source: https://github.com/gathertown/mapmaking
author: Georgios Kaleadis
authorSummary: CTO at Satellytes
---

## Introduction
A few weeks ago, we introduced [Gather](https://www.gather.town/) at Satellytes to make our daily remote life more pleasant and to fight [Zoom Fatigue](https://en.wikipedia.org/wiki/Zoom_fatigue). Gather describes itself as a **_"platform that combines a 2D game-like interface with video-chat"_**. 

> ![Official gather screenshot with people hanging around](images/gather/gather-example-official.png)

Reviewer: I think I would push this back a little bit. I personally found this hindering me getting into the article.

Author: Thanks, I moved it down to the entire introduction. I still want to give a first image to the reader's mind.

![Screen Shot 2021-12-22 at 15 47 13](https://user-images.githubusercontent.com/1701755/147110523-df17ed03-cf41-4f92-9f11-4767c7d45e97.png)

_Official gather screenshot with people hanging around_

> It might look like a game, but it's only the optics. Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.

Reviewer: ```suggestion
At first glance it might look like a game. But that's only the visual presentation chosen by gather (although you can run go-karts, if you like to 😄). The core use-cases are actually all focused around the office live. 

Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.
```
```suggestion
It might look like a game, but it's only the optics. Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.
```

Author: what happened with your first suggestion? I see a lot of html mixed in ?

![Screen Shot 2021-12-22 at 15 51 46](https://user-images.githubusercontent.com/1701755/147111170-ccc613bc-895e-43e7-a8d8-6079ac3c2879.png)

This sounds a lot like [Second Life](https://en.wikipedia.org/wiki/Second_Life). Probably the earliest, rather forgotten, type of Metaverse out there. The thing that makes Gather much different is the 2D appearance and the limited interactions. It's not trying to mimic the real world at all. That keeps the focus on the human interactions, which suffered so much during the COVID-19 pandemic when people were forced to move into the home office.

## How we use Gather
We started using Gather exactly six weeks ago. Your first task is to create your own space. Luckily, they offer plenty of pre-made spaces to pick from. I created a virtual office space, where we could sit at desks or meet at the bar in a kitchen.

![](images/gather/gather-templates.png)
_Pick an existing gather space from plenty of templates_

From the beginning, I forced myself to keep Gather running every day to act as a role model. Occasionally, I would invite someone from the team to try out Gather with me. 

### All-hands meetings

> Once I reached a "critical mass" of people using Gather, I decided to switch our all-hands meeting on Mondays from Zoom to Gather. We would run our meeting on a small, sunny island. That was also the first time I had to use the [mapmaker](https://support.gather.town/help/mapmaker). Because everything is possible in the Metaverse I decided to build a suburban station to connect the island with our office.

Reviewer: ```suggestion
Once we reached a "critical mass" of people using Gather, I decided to switch our all-hands meeting on Mondays from Zoom to Gather. We would run our meeting on a small, sunny island. That was also the first time I had to use the [mapmaker](https://support.gather.town/help/mapmaker). Because everything is possible in the Metaverse I decided to build a suburban station to connect the island with our office.
```

Author: thanks

![](images/gather/gather-smm.jpg)
_Our Satellytes folks gathering for our all-hands meeting on a Monday_

> The actual meeting is comparable to Zoom or Meet. We broadcast some news and people can share thoughts and ask questions. The time before and after that meeting is where the Metaverse shines ✨. 

Reviewer: I personally think the existence of broadcasting spots is even one of the more interesting / useful details. 
I wondered in the beginning myself, how the spatial audio thing would work there

Author: You are right. Actually we don't even need those because meeting rooms allow to speak to everyone else too. The spotlight is rather to reach everyone in every room. I misused that. I check if I can rewrite this part.

This feeling is the promise of the Metaverse. It brings back the informal human interactions and spontaneity. You don't have to create a video meeting upfront or send some specific invitation. Who would do that in real life? Instead, you just walk over to the desk of your colleague and start talking.  

![](images/gather/gather-pair.gif)

_Our colleagues Jesús & Felix having a pair programming session_

### Random bumps
When you walk to someone, while approaching, the video and audio slow fade in. That technique is called "spatial audio technology" and mimics the way we bump into other people in real life, where someone's voice is louder while you are near them, and softer as you walk away. This is wonderful because it feels so familiar and natural.

<video preload autoplay loop width="100%">
  <source src="images/gather/video.mp4" type="video/mp4" />
  <source src="images/gather/video.mp4" type="video/mp4" />
</video>

_Official Gather video from [gather.town](http://www.gather.town)_

> ### Daily

Reviewer: "Daily routine"? "Hanging around"? 
At first I though about the scrum artifact, when I read "Daily", tbh

Author: good point 🙏

While we can gather everyone for some all-hands easily, we have the biggest struggle to motivate people to stay online throughout the day. A few colleagues and I are online most of the day. The other ones are online occasionally or just for the invited events. Sadly, this means our world looks very empty, very often.

![](images/gather/gather-sy-current-world.png)

_Our Satellytes space often looks that empty_

What can we do about that? When I ask people, I get various feedback:

+ I fear I can't work focused enough because people will disturb me all the time
+ I don't trust Gather that the microphone and video is truly switched off every time I focus on another window
+ I'm intimidated with Gather as I don't play games often
+ I just forget to start Gather on the morning

> 

Reviewer: From my personal view: "It doesn't integrate in any way with my work routing / other tools" or "It requires me to spend too much focus on the app itself (the effect 'bumping' into someone is rather an active one)"

Author: thanks I will add those issues

**Do I need to start Gather every day?** <br>
_Yes, of course. Otherwise, you can't participate in the Metaverse and people will miss you_

**Can I close the Gather application?** <br>
_Ideally, you keep it running, but if you have some client meeting through a different channel you might want to close it to prevent any interruptions_

**We are physically in the office, but others are not. Can we merge reality with the Metaverse?** <br>
_That's tricky because we are dealing with the crossing of two realities. Everyone or at least one person in the physical have to join with their avatar to create a bridge between the two worlds_

There are probably many more questions to clarify. I think I will create a living netiquette for Satellytes in 2022.
I hope that document will help people to adapt Gather more easily. 

## Conclusion
Satellytes did not start as a remote-only company more than three years ago. People enjoyed going to the office and talking to each other. That's why our folks had an actual pain when we were forced to work remotely. On the other side, everyone feels the power of remote work, that we want to embrace in the future. 

> The Metaverse seems to be the glue we have been missing all the time. Gather is an excellent way to dive into the Metaverse. I got a big fan of Gather in the past weeks. I'm happily tinkering around with their [mapmaker](https://github.com/gathertown/mapmaking) and constantly improving our virtual space because I really want our folks to enjoy the Metaverse. Let's see where we are heading with that experiment in 2022. 

Reviewer: ```suggestion
The Metaverse seems to be the glue we have been missing all the time. Gather is an excellent way to dive into the Metaverse. I became a big fan of Gather in the past weeks. I'm happily tinkering around with their [mapmaker](https://github.com/gathertown/mapmaking) and constantly improving our virtual space because I really want our folks to enjoy the Metaverse. Let's see where we are heading with that experiment in 2022. 
```

Author: thanks

Author: I think the parsing from github is broken 😂 The highlighted words are escaped and not displayed with the adequate highlight.

![Screen Shot 2021-12-22 at 15 53 47](https://user-images.githubusercontent.com/1701755/147111435-3855e63a-3309-487e-bec9-cbfe6d554b65.png)
